src/v2/kubernetes_cluster/proof/network.rs

```diff
@@ -251,6 +251,64 @@ pub proof fn lemma_always_every_in_flight_req_msg_has_different_id_from_pending_
     init_invariant::<ClusterState>(spec, self.init(), stronger_next, invariant);
 }
 
+pub open spec fn every_in_flight_req_msg_has_different_id_from_all_pending_req_msg(controller_id: int) -> StatePred<ClusterState> {
```

Comment:
Can we follow the naming of `every_in_flight_req_msg_has_different_id_from_pending_req_msg_of` and name this one as `every_in_flight_req_msg_has_different_id_from_pending_req_msg_of_every_ongoing_reconcile`?